metadata-ingestion/src/datahub/cli/delete_cli.py

```diff
@@ -148,14 +162,23 @@ def delete(
     registry_id: str,
     dry_run: bool,
     only_soft_deleted: bool,
+    container: str,
```

Comment:
I don't think we want this additional option - we should just do the right thing when `urn` is of type container